kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -23,6 +26,13 @@ def __init__(self, J, shape, Q=1, T=None, max_order=2, average=True,
         self.vectorize = vectorize
         self.out_type = out_type
         self.backend = backend
+        self.complex_input = complex_input
+
+        warnings.warn("The average argument is deprecated and will be "
+                        "removed in version 0.3." 
```

Comment:
0.4.

actually this warning should be in the main package.

Reply:
OK, where?